### PR TITLE
Calculate subsets of Array in Julia 

### PR DESCRIPTION
**Pull request by Saviour1001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Used bitmasking to calculate all the subsets of an Array in Julia.

### Checklist

- [ Yes ] I've read the contribution guidelines.
- [ Yes ] I've checked the issue list before deciding what to submit.
- [ Yes ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This PR closes issue #5246

### CODE REVIEW

File: Julia/cp/subsetsOfArray.jl [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#= For any given array there are 2^N subsets. Thus generating each and every subset using BitMasking.
+We follow the binary representation of whole numbers and use them as boolean value if a character in the Set
+will occur or not. We will remove the duplicate subsets and display the non null unique subsets.
+=#
+
+## Function
```
Comment (ankitaggarwal23) [inline Julia/cp/subsetsOfArray.jl:6]: Using multiline comments

File: Julia/cp/subsetsOfArray.jl [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    123
+    3
+    5
+    23
+    13
+    14
+    12
+    1
+Time complexity: O(2^N)
+=#
```
Comment (ankitaggarwal23) [inline Julia/cp/subsetsOfArray.jl]: Using multiline comments and add SC and also add new line at the end

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Add a repl.it link of your code

File: Julia/cp/subsetsOfArray.jl (hunk 1 above)
Comment (Saviour1001) [reply Julia/cp/subsetsOfArray.jl:6]: Could you explain what did you meant by adding repl.it link

File: Julia/cp/subsetsOfArray.jl (hunk 2 above)
Comment (Saviour1001) [reply Julia/cp/subsetsOfArray.jl]: Added the Space Complexity and a new line at the end

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**